src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerActionInvoker.cs

```diff
@@ -45,8 +43,7 @@ public class ControllerActionInvoker : ResourceInvoker, IActionInvoker
             ILogger logger,
             DiagnosticSource diagnosticSource,
             ControllerContext controllerContext,
-            IFilterMetadata[] filters,
-            ObjectMethodExecutor objectMethodExecutor)
+            IFilterMetadata[] filters)
```

Comment:
It would be cleaner and more consistent if *everything* in `Microsoft.AspNetCore.Mvc.Internal` was actually `internal`, so making this one type `internal` would be a step towards that.

Again, it's technically a breaking change and I don't have any data on how much people in the wild might be referencing this type presently. So from a code quality perspective I'd support this change, but I can't really estimate the costs to consumers of doing so.